File: IronVault/UI/ironvault_desktop/lib/models/narudzba.dart
import 'package:ironvault_desktop/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'narudzba.g.dart';

@JsonSerializable()
class Narudzba {
  int? narudzbaId;
  int? korisnikId;
  Korisnik? korisnik;
  DateTime? datumVrijemeNarudzbe;
  bool? status;
  bool? otkazano;

  Narudzba({this.narudzbaId});

  factory Narudzba.fromJson(Map<String, dynamic> json) =>
      _$NarudzbaFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$NarudzbaToJson(this);
}

File: IronVault/IronVault.Model/Models/Narudzba.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class Narudzba
    {
        public int NarudzbaId { get; set; }
        public string? Sifra { get; set; }
        public int KorisnikId { get; set; }

        public Korisnik Korisnik { get; set; }

        public DateTime DatumVrijemeNarudzbe { get; set; }

        public bool? Status { get; set; }

        public bool? Otkazano { get; set; }
    }
}
